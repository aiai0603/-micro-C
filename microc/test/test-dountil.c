Void main(Int n) {
    Int i;
    i=0;
   do{
        print("%d",i);
        i = i+1;
   }  
   until(i>n);



}

