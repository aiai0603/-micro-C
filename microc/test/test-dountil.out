24 19 1 5 25 15 1 13 0 1 1 0 0 12 15 -1 13 0 1 1 11 22 15 -1 13 0 1 1 13 0 1 1 11 0 1 1 12 15 -1 13 0 1 1 11 13 11 10 7 17 16 21 1